sagemaker-python-sdk/dgl_gcn/mxnet_gcn_hypertune.ipynb

```diff
@@ -242,9 +242,9 @@
    "metadata": {},
    "source": [
     "## Wrap-up\n",
-    "Now that we've started our hyperparameter tuning job, it will run in the background and we can close this notebook. Once finished, we can go to console to analyze the result.\n",
+    "Now that we've started our hyperparameter tuning job, it will run in the background. You can close this notebook. When it's finished, you can go to console to analyze the result.\n",
```

Comment:
```suggestion
    "Now that we've started the hyperparameter tuning job, it will run in the background. You can close this notebook. When it's finished, you can go to console to analyze the result.\n",
```